test/specs/runLwcTests.e2e.ts

```diff
@@ -0,0 +1,316 @@
+    const runTestsAction = await lwcTestsSection.getAction('SFDX: Run All Lightning Web Component Tests');
+    expect(runTestsAction).to.not.be.undefined;
+    await runTestsAction!.click();
+
+    // Verify test results are listed on the terminal
+    // Also verify that all tests pass
+    const terminalText = await utilities.getTerminalViewText(workbench, 10);
+
+    const expectedTexts = [
+      `PASS  force-app/main/default/lwc/lwc1/__tests__/lwc1.test.js`,
+      `PASS  force-app/main/default/lwc/lwc2/__tests__/lwc2.test.js`,
+      'Test Suites: 2 passed, 2 total',
+      'Tests:       4 passed, 4 total',
+      'Snapshots:   0 total',
+      'Ran all test suites.'
+    ];
+    expect(terminalText).to.not.be.undefined;
+    await utilities.verifyOutputPanelText(terminalText!, expectedTexts);
+
+    // Verify the tests that are passing are labeled with a green dot on the Test sidebar
+    for (const item of lwcTestsItems) {
+      await utilities.verifyTestIconColor(item, 'testPass');
+    }
+  });
+
+  step('Run All Tests on a LWC via the Test Sidebar', async () => {
+    utilities.log(`${testSetup.testSuiteSuffixName} - Run All Tests on a LWC via the Test Sidebar`);
+    const workbench = utilities.getWorkbench();
+
+    // Click the run test button that is shown to the right when you hover a test class name on the Test sidebar
+    const terminalText = await utilities.runTestCaseFromSideBar(
+      workbench,
+      'LWC Tests',
+      'lwc1',
+      'SFDX: Run Lightning Web Component Test File'
+    );
+
+    const expectedTexts = [
+      `PASS  force-app/main/default/lwc/lwc1/__tests__/lwc1.test.js`,
+      'Test Suites: 1 passed, 1 total',
+      'Tests:       2 passed, 2 total',
+      'Snapshots:   0 total',
+      'Ran all test suites within paths',
+      `${path.join('force-app', 'main', 'default', 'lwc', 'lwc1', '__tests__', 'lwc1.test.js')}`
+    ];
+    expect(terminalText).to.not.be.undefined;
+    await utilities.verifyOutputPanelText(terminalText!, expectedTexts);
+    await utilities.closeAllEditors();
+  });
+
+  step('Run Single Test via the Test Sidebar', async () => {
+    utilities.log(`${testSetup.testSuiteSuffixName} - Run Single Test via the Test Sidebar`);
+    const workbench = utilities.getWorkbench();
+
+    // Hover a test name under one of the test lwc sections and click the run button that is shown to the right of the test name on the Test sidebar
+    const terminalText = await utilities.runTestCaseFromSideBar(
+      workbench,
+      'LWC Tests',
+      'displays greeting',
+      'SFDX: Run Lightning Web Component Test Case'
+    );
+    const expectedTexts = [
+      `PASS  force-app/main/default/lwc/lwc1/__tests__/lwc1.test.js`,
+      'Test Suites: 1 passed, 1 total',
+      'Tests:       1 skipped, 1 passed, 2 total',
+      'Snapshots:   0 total',
+      'Ran all test suites within paths',
+      `${path.join('force-app', 'main', 'default', 'lwc', 'lwc1', '__tests__', 'lwc1.test.js')}`
```

Comment:
Here `lwc1` is run and in WDIO `lwc2` is run.  Am I correct to assume that the tests are identical and it doesn't matter which one is run?

Reply:
Yes! In wdio everything was tricky and random, but the one that always made sense to be run is this one. The tests are identical anyways so all good